CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 ### Added
+
+### Changed
+
```

Comment:
Maybe add the non breaking, silent breaking, ... dub-headers

Reply:
fixed.